lodash/index.d.ts

```diff
@@ -12335,7 +12335,7 @@ declare module _ {
          * @param value The value to check.
          * @return Returns true if value is null, else false.
          */
-        isNull(value?: any): boolean;
+        isNull(value?: any): value is null;
```

Comment:
Thanks! Do you know when it will be part of the next release so we can download it with npm?